student.rb

```diff
@@ -9,4 +10,11 @@ def initialize(id, age, classroom, name = 'Unknown', parent_permission: true)
   def play_hooky
     '¯\\(ツ)/¯'
   end
+
+  attr_reader :classroom
+
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍